src/core/event.js

```diff
@@ -752,65 +753,105 @@ Candy.Core.Event = (function(self, Strophe, $) {
 			 */
 			Message: function(msg) {
 				Candy.Core.log('[Jabber:Room] Message');
+
+				var carbon = false,
+					partnerJid = Candy.Util.unescapeJid(msg.attr('from'));
+
+				if (msg.children('sent[xmlns="' + Strophe.NS.CARBONS + '"]').length > 0) {
+					carbon = true;
+					msg = $(msg.children('sent').children('forwarded').children('message'));
+					partnerJid = msg.attr('to');
```

Comment:
This one is not unescaped like https://github.com/candy-chat/candy/blob/feature/message_carbons/src/core/event.js#L769 is, was that intentional?